### ISSUE TITLE
1.20.1 FORGE Flusk Zombie are ticking and cause "Exception in server tick loop"

### ISSUE THREAD

**Issue by Zarvera** [TheASEStefan/RottedZombieMod#1]
Even Neruina doesn't seem to be helping much, it deletes the ticking Flusk Zombie but the game still crashes soon after with the following log
[crash-2023-11-27_23.58.16-server.txt](https://github.com/TheASEStefan/RottedZombieMod/files/13480300/crash-2023-11-27_23.58.16-server.txt)

**Comment by TheASEStefan** [comment]
Ok, I've taken a proper look at the error and I don't see what you mean? Where does it mention that a flusk zombie is crashing your game? And I've seen both fabric mods and forge ones, what are you making, minecraft mod chicken pot?

**Comment by TheASEStefan** [comment]
> Even Neruina doesn't seem to be helping much, it deletes the ticking Flusk Zombie but the game still crashes soon after with the following log [crash-2023-11-27_23.58.16-server.txt](https://github.com/TheASEStefan/RottedZombieMod/files/13480300/crash-2023-11-27_23.58.16-server.txt)

Ok, I've taken a proper look at the error and I don't see what you mean? Where does it mention that a flusk zombie is crashing your game? And I've seen both fabric mods and forge ones, what are you making, minecraft mod chicken pot?

**Comment by Zarvera** [comment]
Neruina reports about deleting ticking entity "Flusk Zombie" half a second before instance crashes down. This consistently happened several times before i just got up and reduced their spawn rate to 0 through configs, which stopped these crashes specifically. So. that's my only lead. 
As for Fabric - yeah, it's Lazurite mod. Dev decided to tag it as a forge mod just because it can be hooked up to Forge through Synitra Connector (Fabric-Forge bridge). Doubt it's any importance, it's some minor performance optimization mod.

**Comment by Zarvera** [comment]
Yes, it's supposed to prevent crashes by removing the ticking entity, and it's what it usually does. In this case however, what happened several times is 
1. Neruina reports about ticking entity being removed
2. <1sec later instance crashes.
More importantly though, i'm finding out my pack is currently entirely messed up due to, i suppose, some questionable lesser known mods. Several mods misbehave, crash or don't work in a way that's not reported by anyone else. 
So, i think you should just close this issue for now. 99% sure those crashes have nothing to do with your mod. Sorry for bothering. 
I'll post here again when i'll get it all under control and if those crashes will persist then

**Comment by TheASEStefan** [comment]
Ok thank you for your patience and time to share this information with me, I appreciate it that you were pretty dedicated to fixing the error, if you find out what caused it, you can tell me :)
